apps/web/src/pages/TalentNominations/NominateTalent/components/Success.tsx

```diff
@@ -0,0 +1,107 @@
+import { useIntl } from "react-intl";
+import CheckCircleIcon from "@heroicons/react/24/outline/CheckCircleIcon";
+import { ReactNode } from "react";
+
+import { FragmentType, getFragment, graphql } from "@gc-digital-talent/graphql";
+import { commonMessages } from "@gc-digital-talent/i18n";
+import { Link } from "@gc-digital-talent/ui";
+
+import useRoutes from "~/hooks/useRoutes";
+
+import useCurrentStep from "../useCurrentStep";
+import SubHeading from "./SubHeading";
+
+const NominateTalentSuccess_Fragment = graphql(/* GraphQL */ `
+  fragment NominateTalentSuccess on TalentNomination {
+    submittedAt
+    nominee {
+      firstName
+    }
+    talentNominationEvent {
+      community {
+        name {
+          localized
+        }
+      }
+    }
+  }
+`);
+
+interface SuccessProps {
+  successQuery: FragmentType<typeof NominateTalentSuccess_Fragment>;
+}
+
+const Success = ({ successQuery }: SuccessProps) => {
+  const intl = useIntl();
+  const paths = useRoutes();
+  const { current } = useCurrentStep();
+  const talentNomination = getFragment(
+    NominateTalentSuccess_Fragment,
+    successQuery,
+  );
+
+  if (current || !talentNomination.submittedAt) {
```

Comment:
How can I bring up the success page?  I manually set a `submitted_at` value and entered all the submitted steps in the array.

Reply:
Looks likke this was a bug in the redirect. The instructions page was taking over. After the change to how that works, this should be fixed [fix redirect to first unsubmitted step](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12972/commits/6479dc1838007fa626fec2bf90db6e55143f107a)

Reply:
This looks better now.  :+1: There's a small visual bug in the stepper though: there's a down-stem on the last dot.  Is there a missing option to set this as the bottom dot?
![image](https://github.com/user-attachments/assets/e1508f35-2eb9-4294-bb05-5e01699d985d)

Reply:
Huh, wonder why this just showed up here :sweat_smile: Anyway, fixed in [fix stepper styles](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12972/commits/13b3fa80768aa888f755682ad93f46285ed7761a)